src/pages/YoutubeHome/YoutubeHome.page.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Nav from '../../components/YoutubeNav/YoutubeNav.component';
+import Grid from '../../components/YoutubeGrid/YoutubeGrid.component';
+import { StyledTitle } from './YoutubeHome.styles';
+
+function Home() {
+  return (
+    <div>
```

Comment:
You could use a React Fragment here to avoid divs that don't provide anything to the layout.

Reply:
I'll address this change in challenge_3 pr